builder/builder/utils.go

```diff
@@ -156,8 +156,11 @@ func EmitBoltDemoEvent(message string) {
 
 func CalculateMerkleMultiProofs(
 	payloadTransactions types.Transactions,
-	constraints types.HashToConstraintDecoded,
+	HashToConstraintDecoded types.HashToConstraintDecoded,
```

Comment:
why not `constraints`? In general I think shorter variables improve readability

Reply:
Just because `constraints` is the name I've given to the constraints array shortly after, so I've changed this to something that resembles more closely its type. However this is not set in stone! Any suggestion is welcome